pom.xml

```diff
@@ -196,6 +196,7 @@
     <maven.resolver.version>1.7.3</maven.resolver.version>
     <maven.version>3.8.4</maven.version>
     <mockito.version>4.3.1</mockito.version>
+    <nessie.antlr.version>4.9.2</nessie.antlr.version>
```

Comment:
since we still use `antlr4-maven-plugin` we need to keep the `antlr.version` property, but does it mean we need to keep those versions in sync or whats the rule for updating one of the two in the future (i.e. `major.minor` version needs to stay the same)?

if there is any relationship we should imo move this line next to the antlr version and add a comment there.

Reply:
Yea - it's not strictly required and this property shall go away.
I had to add it to try the version-bump-scenario.
Will remove this property soon.